### DISCUSSION TITLE
List of error codes.

### DISCUSSION THREAD

**Discussion by pnearing** [AsamK/signal-cli#1438] [Q&A]
Hi, am developing a command line signal client, and I was wondering if anyone knew where I could find a list of error codes that signal returns in different circumstances.  For example, When linking an account, you can get a -1 error code, meaning the account is already linked; Or your can get a -3 error code, meaning the link operation timed out; But nowhere have I seen what -2 error code means, which theoretically should exist.  If anyone can point me in the right direction, that'd be awesome. Thanks in advance.

Peter N.

**Comment by AsamK** [comment]
The error codes are documented in the man page.
https://github.com/AsamK/signal-cli/blob/master/man/signal-cli.1.adoc#exit-codes

**Comment by pnearing** [comment]
I'm sorry, but I'm looking for error codes generated by the protocol. Thank you though. On Feb 4, 2024 12:24, Sebastian Scheibner ***@***.***> wrote:
The error codes are documented in the man page.
https://github.com/AsamK/signal-cli/blob/master/man/signal-cli.1.adoc#exit-codes
